docs/source/dataframe-create.rst

```diff
@@ -58,29 +58,39 @@ and not actively maintained; use at your own risk.
 
 .. code-block:: Python
 
-   >>> from castra import Castra
-   >>> c = Castra(path='/my/castra/file')
-   >>> df = c.to_dask()
+   >>> from castra import Castra # doctest: +SKIP
+   >>> c = Castra(path='/my/castra/file') # doctest: +SKIP
+   >>> df = c.to_dask() # doctest: +SKIP
```

Comment:
I don't think you need to add `# doctest: +SKIP` to every line in a code block. I think one is enough.